ex/get_user_info.pl

```diff
@@ -0,0 +1,9 @@
+use Modern::Perl;
+use Facebook::OpenGraph;
+
+# fetching public information about given objects
```

Comment:
Maybe it's not a good idea to include an example to fetch public information without access_token.
After v2.0, we will need access_token to fetch those public information, too.
And v1.0 is going to be deprecated shortly.
I was going to remove this kind of synopsis on next version.